AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -261,14 +261,15 @@ def AtomEditorComponents_Light_AddedToEntity():
                 light_component.set_component_property_value(
                     AtomComponentProperties.light('Attenuation radius Mode'), ATTENUATION_RADIUS_MODE[radius_mode])
                 general.idle_wait_frames(1)
+                current_radius_mode = light_component.get_component_property_value(
+                    AtomComponentProperties.light('Attenuation radius Mode'))
                 test_attenuation_mode = (
                     f"Attenuation radius Mode set to {radius_mode}",
                     f"P1: Attenuation radius Mode failed to be set to {radius_mode}")
-                Report.result(test_attenuation_mode, light_component.get_component_property_value(
-                    AtomComponentProperties.light('Attenuation radius Mode')) == ATTENUATION_RADIUS_MODE[radius_mode])
+                Report.result(test_attenuation_mode, current_radius_mode == ATTENUATION_RADIUS_MODE[radius_mode])
 
                 # 11. Edit the Attenuation radius Radius parameter (explicit only).
-                if radius_mode == 'explicit':
+                if current_radius_mode == ATTENUATION_RADIUS_MODE['explicit']:
```

Comment:
if ATTENUATION_RADIUS_MODE[radius_mode] in
don't rely on the test response to branch here either

Reply:
Updated